File: Java_Programming/Competative/Assignments/Assignment_3/program2.java
package Competative.Assignments.Assignment_3;
//write a program which accept the number from user and print  factors of that number.
//Input : 24
//Output : 1 2 4 6 8 12 

import java.util.*;

class DisplayX
{
    public void  DisplayFactor(int iNo)
    {
        int i = 0;

        if(iNo <= 0)
        {
            iNo = -iNo;
        }

        for(i = 1; i <= iNo/2; i++)
        {
            if((iNo % i) == 0)
            {
                System.out.print(i);
            } 
        }
    }
}

class program2 
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        DisplayX dobj = new DisplayX();

        dobj.DisplayFactor(iValue);

    }
}

File: C_Programming/Competative/Assignments/Assignment_4/program2.c
//Write a program which accepts the number from user and  display its factors in decreasing order.

//Input : 12
//Output : 6 4 3 2 1

#include<stdio.h>

int FactRev(int iNo)
{
    int iCnt = 0;

    for(iCnt = (iNo / 2); iCnt >= 1; iCnt--)
    {
        if((iNo % iCnt) == 0)
        {
            printf("%d\n",iCnt);
        }
    }
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter the number : \n");
    scanf("%d", &iValue);

    iRet = FactRev(iValue);

    return 0;
}